src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -174,8 +166,8 @@ public List<ModificationInfos> getModifications(UUID groupUuid, boolean onlyMeta
 
     @Transactional(readOnly = true)
     // Need a transaction for collections lazy loading
-    public List<ModificationInfos> getModification(UUID modificationUuid) {
-        return networkModificationRepository.getModifications(List.of(modificationUuid));
+    public List<ModificationInfos> getNetworkModifications(Set<UUID> networkModificationUuid) {
```

Comment:
Rename also `getModifications` by `getNetworkModifications` for a group

Reply:
Done